admiral/pkg/clusters/types.go

```diff
@@ -268,105 +179,15 @@ func (dh *DependencyHandler) Deleted(obj *v1.Dependency) {
 }
 
 func (gtp *GlobalTrafficHandler) Added(obj *v1.GlobalTrafficPolicy) {
-	log.Infof(LogFormat, "Added", "trafficpolicy", obj.Name, gtp.ClusterID, "received")
-
-	var matchedDeployments []k8sAppsV1.Deployment
-	var matchedRollouts []argo.Rollout
-
-	//IMPORTANT: The deployment/Rollout matched with a GTP will not necessarily be from the same cluster. This is because the same service could be deployed in multiple clusters and we need to guarantee consistent behavior
-	for _, remoteCluster := range gtp.RemoteRegistry.RemoteControllers {
-		matchedDeployments = append(matchedDeployments, remoteCluster.DeploymentController.GetDeploymentByLabel(obj.Labels[common.GetGlobalTrafficDeploymentLabel()], obj.Namespace)...)
-		if gtp.RemoteRegistry.AdmiralCache.argoRolloutsEnabled {
-			matchedRollouts = append(matchedRollouts, remoteCluster.RolloutController.GetRolloutByLabel(obj.Labels[common.GetGlobalTrafficDeploymentLabel()], obj.Namespace)...)
-		}
-	}
-
-	deployments := common.MatchDeploymentsToGTP(obj, matchedDeployments)
-	rollouts := common.MatchRolloutsToGTP(obj, matchedRollouts)
-
-	if len(deployments) != 0 {
-		for _, deployment := range deployments {
-			err := gtp.RemoteRegistry.AdmiralCache.GlobalTrafficCache.Put(obj, &deployment)
-			if err != nil {
-				log.Errorf("Failed to add nw GTP to cache. Error=%v", err)
-				log.Infof(LogFormat, "Added", "trafficpolicy", obj.Name, gtp.ClusterID, "Failed")
-			}
-		}
-	}
-
-	if len(rollouts) != 0 {
-		for _, rollout := range rollouts {
-			err := gtp.RemoteRegistry.AdmiralCache.GlobalTrafficCache.PutRollout(obj, &rollout)
-			if err != nil {
-				log.Errorf("Failed to add new GTP to cache. Error=%v", err)
-				log.Errorf(LogErrFormat, "Added", "trafficpolicy", obj.Name, gtp.ClusterID, "Failed")
-			}
-		}
-	}
-
-	if len(deployments) == 0 && len(rollouts) == 0 {
-		log.Infof(LogErrFormat, "Added", "trafficpolicy", obj.Name, gtp.ClusterID, "Skipping, no matched deployments/rollouts")
-	}
-
+	log.Infof(LogFormat, "Added", "globaltrafficpolicy", obj.Name, gtp.ClusterID, "received")
```

Comment:
Will this event also update the destination rule? or it will update the cache and deployment for the identity will update destination rule?

Reply:
It will update the GTP cache only and the DR is updated as part of SE generation. The actual triggering of SE generation on GTP updates will be implemented separately. 
This PR basically removes active querying of GTPs from all clusters (which makes no sense) before SE generation which was adding anywhere from 30 to 40 seconds when a deployment/rollout events happen.

Reply:
Trying to understand the significance of Added, Updated, Deleted functions in GlobalTrafficHandler. I see we are just logging, will it be doing more work in the future?

Reply:
Yes, in near future we will trigger SE generation upon GTP events, for now they are just placeholders.